### ISSUE TITLE
How to configure for hosted Jira? 

### ISSUE THREAD

**Issue by tbernaerd** [dimitri-koenig/alfred-jira-workflow#3]
Hi,

I'm having trouble getting this to work on a hosted (atlassian.net) jira install. How should I do the auth config? Specifically the "where" field? 

I tried with just "https://ourdomain.atlassian.net/" but that doesn't work.

Your confluence workflow does work as expected with https://ourdomain.atlassian.net/wiki in the Keychain.

Many thanks!

**Comment by dimitri-koenig** [comment]
Which version of Alfred do you use? If you updated/installed version 3, please download the newest source cause I made a minor adjustment this morning to get this jira workflow up and running again, after it broke with the update from version 2 to version 3.

**Comment by tbernaerd** [comment]
Hi,

I was using latest version. Think I have found the issue. We have both Confluence and Jira, and they run on the same URL. Confluence was working only because I did it first. What should I enter here to get both confluence and jira to work? 

![image](https://cloud.githubusercontent.com/assets/6125167/17139841/4baa637a-5346-11e6-9cb7-891e3ff493b1.png)

Two entries with the same URL/credentials and different comments does not seem to work. 

Many thanks for your response and the package in the first place.
